project/assets/common.less

```diff
@@ -0,0 +1,32 @@
+h3 {
+  margin-top: 0;
+}
+
+input {
+  border: none;
+  border-bottom: 1px solid lightgrey;
+  font-size: 18px;
```

Comment:
Please use a class with separate structure/skin. The styling(eg margin-top, margin-left) doesn't seem to be generic enought so that it is applied to all input fields